src/NHibernate.Validator/Properties/DefaultValidatorMessages.resx

```diff
@@ -135,6 +135,9 @@
   <data name="validator.email" xml:space="preserve">
     <value>not a well-formed email address</value>
   </data>
+  <data name="validator.enum" xml:space="preserve">
+    <value>must correspond to Enum definition</value>

```

Comment:
`must match the enumeration definition` would sound better in my opinion.

Reply:
Google can have it quite wrong, and that is clearly the case for the German (.de) one. I am French so for the french one better use what I propose than Google.

The .Net resources logic cause it to display the neutral version (English in our case) if the localized version lacks the resource string. I think we should use that behavior rather than providing some auto-translated and non verified translation.

Personally, when I spot a pseudo French text issued by such an automated translation (be it Google, Microsoft, Facebook or others), I just disregard it as garbage. It is too often non-sens and sometimes it just means the precise opposite of what it should, being terribly misleading. (I have seen that very frequently in technical documentation, sometime even in those manually translated but by translators not knowledgeable of the field... And automated translations are worst.)

Reply:
Lets remove all except En & Fr 
If somebody will need this on other languages, it will be additional PR